packages/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -30,7 +38,15 @@ registerInternalPlugin('calendar', Calendar, {
         name: 'transformMeetingParticipants',
         direction: 'inbound',
         test(ctx, response) {
-          return Promise.resolve(has(response, 'body.encryptedParticipants'));
+          return Promise.resolve(
+            has(response, 'body.encryptedParticipants') &&
+              !(
+                response.options &&
+                response.options.service === 'calendar' &&
+                response.options.method === 'GET' &&
+                response.options.resource === 'schedulerData'
+              )
+          );
         },
         extract(response) {
           return Promise.resolve(response.body);
```

Comment:
we can just return the data rather then promise.resolve here

Reply:
@arun3528 the extract method should return a promise, otherwise it will pop up this JS error:

webex.min.js:2 Uncaught (in promise) TypeError: e.extract(...).then is not a function
    at webex.min.js:2:1379558
    at async Promise.all (/webappng/sites/schedule01/meeting/index 10)
    
I also checked webex-core.js file, within transform(direction, object) {...} method, it call extract method like this:

return (
            p
              .extract(object)
              // eslint-disable-next-line max-nested-callbacks
              .then((target) => ({
                name: p.name,
                target,
              }))
          );
          
    So we need this method return a promise.

Reply:
k